packages/docs/docs/tutorial/part2.md

```diff
@@ -0,0 +1,64 @@
+---
+sidebar_position: 2
+---
+
+# Tutorial Part 2 - Progressive rendering
+
+Let's look at a slightly more complex example, which uses the `<Inline>` component
+to progressively render the result of the LLM invocation as it is generated. This
+allows us to take the current result generated by the LLM and pass it into the next
+invocation of the LLM.
+
+```tsx filename="packages/tutorial/src/part2.tsx"
+import * as LLMx from 'ai-jsx';
+import { Inline } from 'ai-jsx/core/inline';
+import { ChatCompletion, UserMessage } from 'ai-jsx/core/completion';
+import { showInspector } from 'ai-jsx/core/inspector';
+
+const app = (
+  <Inline>
+    <ChatCompletion>
+      <UserMessage>Come up with the name of a mythical forest animal.</UserMessage>
+    </ChatCompletion>
+    {'\n\n'}
+    {(conversation) => (
```

Comment:
I know you explain it in the text below, but I think adding a comment above here saying that `conversation` is the previous output of the LLM helps explain the code. Otherwise it's not clear where this is coming from

Reply:
I prefer to show the code first and explain it second - more consistent that way.